internal/fleetdb/fleetdb.go

```diff
@@ -42,6 +50,57 @@ func NewFleetDBClient(_ context.Context, cfg *app.Configuration) (Client, error)
 	}, nil
 }
 
+func getFleetDBAPIClient(ctx context.Context, cfg *app.FleetDBAPIOptions) (*fleetdb.Client, error) {
+	fmt.Printf("cfg = %+v\n", cfg)
+	if cfg.DisableOAuth {
+		return fleetdb.NewClientWithToken("fake", cfg.Endpoint, nil)
+	}
+
+	// init retryable http client
+	retryableClient := retryablehttp.NewClient()
```

Comment:
I think we should not use the retryablehttp client since it hides away errors, thoughts @DoctorVin ?

Reply:
I'm more inclined to keep it, at least in the immediate term. My opinion is that I don't really care why the other side threw a `5XX` error back to us, we will need to retry (with some reasonable policy) until it either goes through or we decide to give up, at which point we should log/increment metrics/emit an event.